### ISSUE TITLE
Conditional BERT Contextual Augmentation

### ISSUE THREAD

**Issue by aisolab** [aisolab/paper-log#6]
## 어떤 내용의 논문인가요? 👋

- text data augmentation을 BERT의 masked language model을 이용, word와 paradigmatic relation 관계가 있는 word로 replace하는 방식에 대하여 설명함.
- 특히 paradigmatic relation 관계가 있는 word로 replace 시, 원 sentence의 label과 compatibility가 떨어지는 sentence가 만들어 질 수 있으므로, 이를 해결하기위해 label condition을 주는 방법을 설명함.

## Abstract (요약) 🕵🏻‍♂️

***We propose a novel data augmentation method for labeled sentences called conditional BERT contextual augmentation.*** *Data augmentation methods are often applied to prevent overfitting and improve generalization of deep neural network models. Recently proposed contextual augmentation augments labeled sentences by randomly replacing words with more varied substitutions predicted by language model. BERT demonstrates that a deep bidirectional language model is more powerful than either an unidirectional language model or the shallow concatenation of a forward and backward model.* ***We retrofit BERT to conditional BERT by introducing a new conditional masked language model task. The well trained conditional BERT can be applied to enhance contextual augmentation.*** *Experiments on six various different text classification tasks show that our method can be easily applied to both convolutional or recurrent neural networks classifier to obtain obvious improvement.*

## 이 논문을 읽어서 무엇을 배울 수 있는지 알려주세요! 🤔

- paradigmatic relation
- language model의 의미

## 같이 읽어보면 좋을 만한 글이나 이슈가 있을까요?

- [Contextual Augmentation: Data Augmentation by Words with Paradigmatic Relations](https://arxiv.org/abs/1805.06201)

## 레퍼런스의 URL을 알려주세요! 🔗

- https://arxiv.org/abs/1812.06705

**Comment by aisolab** [comment]
https://aisolab.github.io/paper/Conditional-BERT-Contextual-Augmentation/
